zeeguu/core/feed_handler/feed_handler.py

```diff
@@ -11,9 +12,18 @@ def __init__(self, url: str, feed_type: int):
         self.image_url_string = ""
 
     def get_server_time(self, article_date) -> datetime:
```

Comment:
could this be all moved to normalize_to_server_time? the only thing that it does besides that is 
- converts a potential struct_time to datetime (could belong to the normalize)
- returns a default value if something goes wrong in the conversion or normalization I guess? 

and then 'normalize' to server time, could it be called 'convert_to_server_time'? I'm a bit unsure of the 'normalize' appropriateness in this context.

Reply:
Sure, I moved it inside the now called `convert_to_server_time`. What about the other comment of RSS feeds not getting the date? Should we then attempt to use the newspaper?

Reply:
why not! the only thing that I don't understand is why "is lower" means is better :) oh, i see: this is to solve the situation of dates in the future? if we test the earlier date, then if one is in the future, and the other is correct, we'll be using the correct one? that's fine! maybe add a little comment.

Reply:
Okay, that's a fair question but here is my thought process:

- At this point, it's not an article in the future - they are filtered out before.
- If the date is older, then it's probably because the RSS didn't carry any date, and was set datetime.now()
- So if the newspaper gets an older date, it means that it successfully parsed a date that wasn't today.  If it failed, then the date will be more recent than the parsed on and we keep the oldest (when we crawled the article in the first place).

I do get that we could have a situation where the RSS feed gives us a more recent date than the newspaper and then we would overwrite it, essentially it's almost an Heuristic -> **If we find an older date when parsing, we use the oldest date.**, as long as we are consistent, I think it would be OK.

Reply:
That sounds fair enough. Add this explanation as comment in the code, and it should be good! Then ping me when I should merge :)